src/Persistance/RepositoryImplementations/SystemRegisterRepository.cs

```diff
@@ -228,8 +225,52 @@ private static ValueTask<RegisteredSystem> ConvertFromReaderToSystemRegister(Npg
     }
 
     /// <inheritdoc/> 
-    public Task<bool> CreateClient(string clientId)
+    public async Task<bool> CreateClient(string clientId)
     {
-        throw new NotImplementedException();
+        Guid insertedId = Guid.Parse(clientId);
+
+        const string QUERY = /*strpsql*/@"
+            INSERT INTO altinn_authentication.maskinporten_client(
```

Comment:
i think the schema name can be specific because altinn_authentication is broad. we might have new tables in this database in future may be

Reply:
Ok, done, will be visible in the new PR